maquetado/index.html

```diff
@@ -7,28 +7,83 @@
     <meta name="author" content="Wolox" />
     <link rel="stylesheet" type="text/css" href="application.css" />
   </head>
-  <body>
-    <div class="book-cover">
-      <div class="image-container">
-        <img class="book-image" src="image/book-cover.png" alt="Book Image" />
-      </div>
-      <div class="book-content-title">
-        <div class="book-detail line-separator">
-          <h2 class="book-title">Titulo del Libro</h2>
-          <p class="book-genre">(género)</p>
-        </div>
-        <div class="book-detail">
-          <h3 class="book-detail-title">Autor del libro:</h3>
-          <p class="book-description">Nombre del autor del libro</p>
-        </div>
-        <div class="book-detail">
-          <h3 class="book-detail-title">Editorial:</h3>
-          <p class="book-description">Nombre de la editorial</p>
-        </div>
-        <div class="book-detail">
-          <h3 class="book-detail-title">Año de publicación:</h3>
-          <p class="book-description">Año de publicación</p>
-        </div>
+  <body class="container-books">
+    <nav class="navbar">
+      <img class="wolox-image" src="image/LogoWolox.png" alt="Book Image" />
+      <a class="logout">Logout</a>
```

Comment:
Even if you are not going to redirect or use this button yet it is a good idea to set href attribute anyway, like this 

```suggestion
      <a class="logout" href="#">Logout</a>
```